spotify_dl/youtube.py

```diff
@@ -119,6 +120,8 @@ def set_tags(temp, filename, kwargs):
         )
 
     song_file["genre"] = song.get("genre")
+    if song.get("tempo") != None:
+        song_file["bpm"] = str(song.get("tempo"))
```

Comment:
```suggestion
    if song.get("tempo") is not None:
        song_file["bpm"] = str(song.get("tempo"))
```
`is Not None`  is preferred for objects, see https://note.nkmk.me/en/python-none-usage/

alternatively you can also do  but there are some nuances around it, as mentioned in the above link

```python
    if song.get("tempo"):
        song_file["bpm"] = str(song.get("tempo"))
```